models/app.model.js

```diff
@@ -107,3 +107,17 @@ exports.findCommentByCommentId = (id) => {
       }
     });
 };
+
+exports.readAllUsers = () => {
+  return db
+    .query("SELECT * FROM users")
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Users are not found" });
+      }
+      return rows;
+    })
```

Comment:
There's no reason to end up with no rows - you also don't have a test that covers this anyway. So I would remove. 

Also - the catch you have below would catch this error and return the promise.reject you have in there anyway - so it's not doing anything